cli/commands/update.go

```diff
@@ -139,5 +135,7 @@ func HandleUpdate(app *kingpin.Application) {
 	update := app.Command("update", "Update the package version or configuration for this DC/OS service").Action(updateCmd.UpdateConfiguration)
 	update.Flag("options", "Path to a JSON file that contains customized package installation options").StringVar(&updateCmd.OptionsFile)
 	update.Flag("package-version", "The desired package version").StringVar(&updateCmd.PackageVersion)
-	update.Flag("status", "View status of this update").BoolVar(&updateCmd.Status)
+
+	status := update.Command("status", "View status of the current update").Alias("show").Action(updateCmd.PrintStatus)
```

Comment:
Something funky is going on here. When I run the status command, it prints out the "Update started" message:
```
➜  helloworld git:(cli-plan-status) ✗ dcos hello-world update status
2017/06/02 17:17:48 Update started. Please use `dcos hello-world --name=hello-world service update status` to view progress.
2017/06/02 17:17:48 HTTP GET Query for http://sunil-51i-elasticl-otjoj8l120s3-916731190.us-west-2.elb.amazonaws.com/service/hello-world/v1/plans/deploy failed: 404 Not Found
➜  helloworld git:(cli-plan-status) ✗ dcos hello-world update status
2017/06/02 17:18:49 Update started. Please use `dcos hello-world --name=hello-world service update status` to view progress.
2017/06/02 17:18:49 deploy (STARTING)
├─ hello (COMPLETE)
│  └─ hello-0:[server] (COMPLETE)
└─ world (STARTING)
   ├─ world-0:[server] (COMPLETE)
   └─ world-1:[server] (STARTING)
```